packages/builder/src/components/automation/AutomationBuilder/FlowChart/FlowItemHeader.svelte

```diff
@@ -69,12 +111,32 @@
       <div class="iconAlign">
         {#if isTrigger}
           <Body size="XS"><b>Trigger</b></Body>
-          <Body size="XS">When this happens:</Body>
         {:else}
           <Body size="XS"><b>Step {idx}</b></Body>
-          <Body size="XS">Do this:</Body>
         {/if}
-        <Detail size="S">{block?.name?.toUpperCase() || ""}</Detail>
+        <input
+          placeholder="Enter some text"
+          name="name"
+          autocomplete="off"
+          value={automationName}
+          on:input={e => {
+            automationNameError = getAutomationNameError(e.target.value)
+            automationName = e.target.value
+            if (!automationNameError) {
+              automationNameError = false // Reset the error when input is valid
+            }
```

Comment:
This statement doesn't do anything - it's checking if something is falsey, then setting it to false if it is. I think we can just remove this if statement.